packages/wrangler/src/index.ts

```diff
@@ -588,17 +588,66 @@ export function createCLIParser(argv: string[]) {
 					describe: "The name of your worker",
 					type: "string",
 				})
+				.command(
+					"rollback <deployment-id>",
+					"🔙 Rollback a deployment",
+					(rollbackYargs) =>
+						rollbackYargs.positional("deployment-id", {
+							describe: "The ID of the deployment to rollback to",
+							type: "string",
+							demandOption: true,
+						}),
+					async (rollbackYargs) => {
+						const firstHash = rollbackYargs.deploymentId.substring(
+							0,
+							rollbackYargs.deploymentId.indexOf("-")
+						);
+						if (
+							!(await confirm(
+								`This deployment ${firstHash} will immediately replace the current deployment and become the active deployment across all your deployed routes and domains. However, your local development environment will not be affected by this rollback.`
```

Comment:
@JacobMGEvans  following our discussion earlier, let's add the same note here as in the dash confirmation modal. so to the end of this please add:

"Note: Rolling back to a previous deployment will not rollback any of the bound resources (Durable Object, R2, KV, etc.)."